lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip/search-facet-chip.component.ts

```diff
@@ -46,7 +46,6 @@ export class SearchFacetChipComponent {
     onMenuOpen() {
         if (this.menuContainer && !this.focusTrap) {
             this.focusTrap = this.focusTrapFactory.create(this.menuContainer.nativeElement);
-            this.focusTrap.focusInitialElement();
```

Comment:
if you remove this line, the whole sense in having a focusTrab attribute gets lost... the only thing that is left is assigning this property, but not using it...

Reply:
The focusTrap does seem to work even after removing this line. I suspect that is because we are passing it the native element on which the focusTrap is to be initialised (see line num 48, right above this change). The user experience after this change came out to be - 

1. User performs a search (same as before, no change here)
2. Once search results are received, he clicks on one of these (search filters?) chips (same as before, no change here)
3. On clicking on the chip, the focus remains on the chip itself (changed from before, where the focus was getting jumped to the first element in the panel/popover)
4. User presses Tab. The focus is switched to the first element in the popover (changed from before, this focus was received as soon as step#3 was performed)
5. Now the user can keep on pressing 'Tab'. The focus remains within the fields in the popover (same as before, no change here)

Also, we do need to initialise the focusTrap and store it in a const reference. This is because we are using that reference when the popover is closed, to unregister the focusTrap (see [line num 52](https://github.com/Alfresco/alfresco-ng2-components/blob/b0883788338c9534f60c3a08b9cd3cdbc8984b70/lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip/search-facet-chip.component.ts#L52) onwards)